src/app/pages/datasets/components/dataset-management/dataset-management.component.ts

```diff
@@ -308,15 +308,15 @@ export class DatasetsManagementComponent implements OnInit, AfterViewInit, OnDes
 
   loadHaEnabled(): void {
     if (this.systemService.isEnterprise) {
```

Comment:
Is seems like this whole method is unnecessary now.
We basically just want to hide a button on HA systems.
This could be something like:
```
hasHa$ = this.store$.select(selectIsHaEnabled))
```
and then use it via async pipe in template.